python/python/bystro/parent_of_origin/parent_of_origin.py

```diff
@@ -84,6 +84,8 @@ def _test_inputs(self, X: np.ndarray, y: np.ndarray) -> None:
             raise ValueError("Too small of homozygous sample size, (>30)")
         if np.sum(y == 1) < 30:
             raise ValueError("Too small of heterozygous sample size, (>30)")
+        if not np.all(np.isin(y, [0, 1, 2])):
+            raise ValueError("The array contains values other than 0, 1, or 2")
```

Comment:
We should explicitly exclude values less than 0 or which are nan. Bystro genotype dosage matrices encode missing genotypes as -1

Reply:
You mean eliminate them rather than having the method fail?